### PR TITLE
support setting html charset

### PR DESCRIPTION
**Pull request by Lihuanghe**

### CODE REVIEW

File: reporter.js [hunk 1]
```diff
@@ -104,6 +104,7 @@ module.exports = {
 
         function getRenderedHTML() {
             return templates.body
+                .replace('{charset}', opts.htmlcharset||"utf-8")
```
Comment (ivan-vesely) [inline reporter.js]: @Lihuanghe Thanks for your PR. 
Would you please change it with `.replace('{charset}', opts.htmlcharset || 'utf-8')`. Just wanted to keep convention

Comment (Lihuanghe) [inline reporter.js]: @ivan-vesely 
it has been changed it .

Comment (ivan-vesely) [inline reporter.js]: @Lihuanghe No, i mean spaces around `||` and single quote instead of double quote.

Comment (Lihuanghe) [inline reporter.js]: @ivan-vesely 
done. 
see the commit ”keep convention“:  	a362e61

```javacript

          function getRenderedHTML() {
              return templates.body
 -                .replace('{charset}', opts.htmlcharset||"utf-8")
 +                .replace('{charset}', opts.htmlcharset || 'utf-8')
                  .replace('{content}', prepareContent())
                  .replace('{summary}', prepareSummary());
          }
```

**Comment by ivan-vesely** [comment]
Merged, thanks!

**Pull request merged**